CodeStarterCamp_Week3/coffeeShop.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  coffeeShop.swift
+//  CodeStarterCamp_Week3
+//
+//  Created by daekyo on 2023/10/24.
+//
+
+import Foundation
+
+enum Coffee: String {
+    case americano, iceAmericano, cafeLatte, hotChoco, vanillaLatte
+    
+    var name: String {
+        switch self {
+        case .americano: return "Americano"
+        case .iceAmericano: return "Ice Americano"
+        case .cafeLatte: return "Cafe Latte"
+        case .hotChoco: return "Hot Choco"
+        case .vanillaLatte: return "Vanilla Latte"
+        }
+    }
+}
+
+class CoffeeShop {
+    var salesAmount: Int = 0
+    var menuPan: [Coffee: Int] = [:]
+    var pickUpTable: [Coffee] = []
+    var barista: Person = Person(money: 0, name: "")
+    
+    func takeOrderFor(coffee: Coffee) {
+        makeCoffeeFor(coffee: coffee)
+    }
+    
+    func makeCoffeeFor(coffee: Coffee) {
```

Comment:
커피를 만드는 기능이라면 이름을 아래처럼 간단히 할 수 있을 것 같습니다 💭
```suggestion
    func make(coffee: Coffee) {
```

Reply:
네 위쪽에서 말씀하신 것처럼, 가독성을 생각하고 이름을 짓도록 노력해야 겠습니다.